html/browsers/the-window-object/apis-for-creating-and-navigating-browsing-contexts-by-name/open-features-tokenization-left-vs-screenx.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML: window.open `features`: tokenization -- position features `left` vs `screenx`</title>
+<meta name=timeout content=long>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#apis-for-creating-and-navigating-browsing-contexts-by-name">
+
+<!-- user agents are not required to support open features other than `noopener`
+     and on some platforms position and size features don't make sense -->
+<meta name="flags" content="may" />
+
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/PrefixedPostMessage.js"></script>
+<script>
+var windowURL = 'resources/message-opener.html';
+var featuresPrefix = `width=401,height=204,top=0,`;
+
+setup (() => {
+  // Before running tests, open a window using features that mimic
+  // what would happen if the feature tested here were set to 0
+  // for comparison later.
+  var featureString = `${featuresPrefix}left=0`;
+  var prefixedMessage = new PrefixedMessageTest();
+  prefixedMessage.onMessage((data, e) => {
+    e.source.close();
+    runWindowTests(data);
+  });
+  var win = window.open(prefixedMessage.url(windowURL), '', featureString);
+});
+
+function runWindowTests (baselineDimensions) {
+
+  // The last specified feature is used
+  [ 'left=101,left=102',
+    'left=101,screenx=102',
+    'screenx=101,left=102',
+    'screenx=101,screenx=102',
+  ].forEach((features, idx, arr) => {
+    async_test(t => {
+      var prefixedMessage = new PrefixedMessageTest();
+      prefixedMessage.onMessage(t.step_func_done((data, e) => {
+        e.source.close();
+        assert_equals(data.left, 102);
+      }));
+      var win = window.open(prefixedMessage.url(windowURL), '', featuresPrefix + features);
+    }, `${format_value(features)} repeated left/screenx features`);
+  });
+
+  // The last specified feature is used even if the value is 0 or error
+  [ 'left=101,left=0',
```

Comment:
Filed https://github.com/w3c/web-platform-tests/issues/6301 to follow up on this.